test/Dialect/SV/canonicalization.mlir

```diff
@@ -311,3 +311,10 @@ hw.module @MergeAssignments(%a: !hw.array<4xi1>, %clock: i1) -> (d: !hw.array<4x
   }
   hw.output %read : !hw.array<4xi1>
 }
+
+// CHECK-LABEL: @Sampled
```

Comment:
This doesn't test anything.

Reply:
If sampled op doesn't have Pure, canonicalizer cannot remove dead sampled op so this test checks the module body is empty